Makefile.common

```diff
@@ -5,12 +5,12 @@ MOCHA_FILE      ?= build/test-results/test/TEST-$(PROJECT_NAME).xml
 SKIP_TESTS      ?= false
 MAVEN_BATCH     ?= true
 UNAME_S := $(shell uname -s)
-LN=ln
+LN=ln -T
 BUILD_GOOS=linux
 BUILD_GOARCH=amd64
 
 ifeq ($(UNAME_S),Darwin)
-	LN = gln
+	LN = gln -h
```

Comment:
```
gln -h                                                                                                        
gln: invalid option -- 'h'
Try 'gln --help' for more information.
```

Reply:
Sorry, I missed the fact that it switches to GNU ln on mac